manifests/cert.pp

```diff
@@ -41,7 +41,7 @@
   $certchainfile = "${name}_chain.crt",
   $certinterfile = "${name}_inter.crt",
   $keyfile       = "${name}.key",
-  $source        = "puppet://${::secure_server}/modules/ssldata",
```

Comment:
This just didn't work at all, but on our PE 3.2 infrastructure, we had certs deployed across all the masters, which masked the bugs in this code.